### ISSUE TITLE
Maps support broken?

### ISSUE THREAD

**Issue by FelixAkk** [uptechteam/default.wtf#19]
Hi y'all :) lovely addon! Unfortunately it seems that it doesn't work for Google Maps (anymore?). Could this have to do with a URL mismatch? I believe Google recently started dropping `maps.google.com` in favor of `google.com/maps`. Though I'm not sure. Perhaps just updating this line? https://github.com/uptechteam/default.wtf/blob/8e9dd7a186af71fc9c23ad0cccae6c059de24726/utils.js#L62

**Comment by AndriyBas** [comment]
@FelixAkk yes, correct, seems Google dropped "maps.google.com" in favor or "google.com/maps".
I noticed they also did the same for "finance.google.com".
We'll try to fix it in the next release and support these changes. 
If you know of other products that they made this change — let me know please 🙏. Thanks.
